util/foxsi-setup-script-2014.pro

```diff
@@ -126,4 +114,4 @@ save, tlaunch, t_launch, t1_pos0_start, t1_pos0_end, t1_pos1_start, $
 	shift0, shift1, shift2, shift3, shift4, shift5, shift6, rot0, rot1, rot2, rot3, $
 	rot4, rot5, rot6, $
 	detnum0, detnum1, detnum2, detnum3, detnum4, detnum5, detnum6, $
-	file = 'data_2014/flight2014-parameters.sav'
+	file = FOXSIDB+'/data_2014/flight2014-parameters.sav'
```

Comment:
I would recommend we not use .sav files at all. Not very friendly on github or anybody else.